modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -133,31 +143,37 @@ protected void doStart() {
         String networkAlias = getNetworkAliases().get(0);
         proxy = new SocatContainer()
             .withNetwork(getNetwork())
-            .withTarget(8091, networkAlias);
+            // a (random and unused) target port is needed so that the container starts and stays active
```

Comment:
does it actually makes sense to re-map the REST port? It works fine with dynamic mapping, unlike others

Reply:
well, for consistency reasons, I would say yes, but I also see that this `hack` is not great. Is there another way to start the socat container and let it stay active? Maybe just starting it with `bash`?

Reply:
Then I would suggest using `new GenericContainer(TestcontainersProperties.getSocatImage()).withCommand("ps")`, since you're not using `SocatContainer`'s code at all :)